patches/relay-test-utils-internal+11.0.2.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/node_modules/relay-test-utils-internal/lib/testschema.graphql b/node_modules/relay-test-utils-internal/lib/testschema.graphql
+index ce0e0f3..1071e39 100644
+--- a/node_modules/relay-test-utils-internal/lib/testschema.graphql
++++ b/node_modules/relay-test-utils-internal/lib/testschema.graphql
```

Comment:
I assume this file is not needed for bundled package? Just want to make sure that the bundled package won't break because of this patch.

Reply:
Correct, this is only needed for our tests.